src/object/cia301/co_hb_cons.c

```diff
@@ -262,7 +262,7 @@ int16_t CONmtHbConsCheck(CO_NMT *nmt, CO_IF_FRM *frm)
     int16_t    result = -1;
     uint32_t   cobid;
     uint32_t   ticks;
-    uint8_t    nodeid = NULL;
+    uint8_t    nodeid = 0;

```

Comment:
This seems to cause a problem with the build.  Not all libraries support the NULL define for zero.